CONTRIBUTING_ID.md

```diff
@@ -0,0 +1,98 @@
+## Cara berkontribusi
```

Comment:
```suggestion
# Berkontribusi ke WargaBantuWarga

## Cara berkontribusi
```

@Namchee Always start a Markdown document with h1. We can use the same heading from the English document.

Reply:
Solved by ab1903a2ba95a9b39c9afe7594a16ca5a630ed75